### PR TITLE
[source-us-census] fix empty fields after sync

### LINKED ISSUE #45202: [source-us-census] doesn't return any fields
### Connector Name

source-us-census

### Connector Version

0.2.0

### What step the error happened?

During the sync

### Relevant information

_No response_

### Relevant log output

_No response_

### Contribute

- [X] Yes, I want to contribute

### PR DESCRIPTION
**Pull request by topefolorunso**
## What

Fixes the empty records returned by syncing from the source

## How

Defining the schema depending on the `query_params` config

## User Impact

Records will now be synced.

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-integrations/connectors/source-us-census/source_us_census/components.py [hunk 1]
```diff
@@ -132,4 +135,46 @@ def interpret_response(self, response_or_exception: Optional[Union[requests.Resp
...
+    These APIs return data in a non standard format.
+    We create the JSON schemas dynamically by reading the first "row" of data we get.
+
+    In this implementation all records are of type "string", but this function could
+    be changed to try and infer the data type based on the values it finds.
+    """
+
+    config: Config
+
+    def get_json_schema(self) -> Mapping[str, Any]:
```
Comment (girarda) [inline airbyte-integrations/connectors/source-us-census/source_us_census/components.py:159]: @topefolorunso can you add some unit tests to verify this works as expected?

Comment (topefolorunso) [reply airbyte-integrations/connectors/source-us-census/source_us_census/components.py:159]: Tests added now @girarda

**Review by natikgadzhi — APPROVED** [review]

**Pull request merged**